Foundation/NSGeometry.swift

```diff
@@ -74,9 +74,10 @@ extension CGPoint: NSSpecialValueCoding {
             return false
         }
     }
-    
-    var hash: Int {
-        return self.x.hashValue &+ self.y.hashValue
+
+    func hash(into hasher: inout Hasher) {
```

Comment:
The Hashable conformances for CGPoint, CGSize and CGRect are inconsistent with Apple platforms. (See https://bugs.swift.org/browse/SR-10476.)